gradle.properties

```
@@ -0,0 +1,12 @@
+org.gradle.caching=true
+group=io.ballerina.lib
+
+# This is a dummy value as repo contains multiple packages
+version=1.0.0-SNAPSHOT
+deliveryVersion=1.0.0
```

```suggestion
deliveryVersion=1.0.0-SNAPSHOT
```
The version should always be in development version